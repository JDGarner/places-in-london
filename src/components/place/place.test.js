import React from 'react';
import Place from './place';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('<Place /> renders correctly', () => {
  const tree = shallow(
    <Place name="Park"/>
  );

  expect(toJson(tree)).toMatchSnapshot();
});

// it('should call event handler', () => {
//     const value = '2';
//     const onChange = jest.fn();
//     const wrapper = shallow(
//         <Component onChange={onChange} />
//     );
//
//     expect(wrapper).toMatchSnapshot();
//
//     wrapper.find('select').simulate('change', {
//         target: { value },
//     });
//
//     expect(onChange).toBeCalledWith(value);
// });
