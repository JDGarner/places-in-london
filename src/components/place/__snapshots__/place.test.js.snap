// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Place /> renders correctly 1`] = `
<section
  className="place"
>
  Park
</section>
`;
